src/spikeinterface/core/recording_tools.py

```diff
@@ -69,7 +69,7 @@ def _init_binary_worker(recording, file_path_dict, dtype, byte_offest, cast_unsi
 def write_binary_recording(
     recording: "BaseRecording",
     file_paths: list[Path | str] | Path | str,
-    dtype: np.ndtype = None,
+    dtype: np.dtype = None,
```

Comment:
```suggestion
    dtype: np.typing.DTypeLike = None,
```

Reply:
Make sense as we also allow strings to be passed and not only proper objects.